src/v3/src/transformer/phone/phoneChallengeTransformer.ts

```diff
@@ -37,8 +37,8 @@ export const transformPhoneChallenge: IdxStepTransformer = ({ transaction, formB
     reminderElement = {
       type: 'Reminder',
       options: {
-        content: loc('oie.phone.verify.sms.resendText', 'login'),
-        buttonText: loc('email.button.resend', 'login'),
+        content: methodType === 'sms' ? loc('oie.phone.verify.sms.resendText', 'login') : loc('oie.phone.verify.call.resendText', 'login'),
+        buttonText: methodType === 'sms' ? loc('oie.phone.verify.sms.resendLinkText', 'login') : loc('oie.phone.verify.call.resendLinkText', 'login'),
```

Comment:
Same idea here with `methodType` being pulled out. We already treat it as binary (e.g. if it's not "sms" then we assume "voice", so it may make more sense to treat is as a boolean (binary) value also